engine/lite/components-api/src/main/scala/pl/touk/nussknacker/engine/lite/api/interpreterTypes.scala

```diff
@@ -12,7 +12,7 @@ object interpreterTypes {
 
   case class SourceId(value: String)
 
-  case class ScenarioInputBatch[Input](value: List[(SourceId, Input)])
+  case class ScenarioInputBatch[Input](value: List[(SourceId, Input, Map[String, String])])
```

Comment:
Maybe we should introduce here a new case class: ScenarioInputValue with 3 fields?

Reply:
I think this is a good idea :+1: